### ISSUE TITLE
Select area with mousedown-mouseup

### ISSUE THREAD

**Issue by OoDeLally** [gTile/gTile#97]
Thanks for this amazing app!
Currently selecting an area takes two mouse clicks (1st to select the starting tile, and 2nd to select the expansion of the rectangle).
Would it be possible to bind the starting tile to `mousedown` and validate the area with `mouseup`?
So that it takes only one click to select an area.
Thanks!

**Comment by scherepanov** [comment]
Yes, that would be a good idea. That would be more in-line with mainstream GUI behaviour.
Problem is that historically someone many years ago implemented it this way. Now changing it would be forcing this change to a user base... So it never got changed.
And, original mouse-down + mouse+down interface is not that bad, as it also tracks mouse pointer and reflect it on a screen.

**Comment by cwh1te** [comment]
@scherepanov sorry to necromance this issue, but wouldn't it be possible to implement this behavior only for mouse-down, drag (more than one square), and mouse-up? So if you mouse down and then up the normal behavior applies, but if you move the mouse some distance between then this new behavior would be desirable.
I haven't looked at the code in a couple years so I'm probably forgetting something that prevents this from working...

**Comment by OoDeLally** [comment]
We could even provide it as a optional behavior (a checkbox in the preference panel).
So we dont force this change to the user base.

**Comment by OoDeLally** [comment]
This seems to only be a partial solution. If we do this, assigning a window to one (1) tile will still require two clicks. Right?
I would rather have a checkbox in the settings to actually switch between the two behaviors.
